text/nodejs/0013-install-bom.md

```diff
@@ -61,36 +61,70 @@ the following fields for each module:
 All of these are fields that we aim to support per [the Paketo BOM
 RFC](https://github.com/sophiewigmore/rfcs/blob/8b1e8c9ed6201313f47c5897223cbffb265e96ed/text/0028-bill-of-materials.md)
 
-### Buildpack Additions
+In order to support this tool in both the online and offline cases, we will
+pre-compile the tool with it's dependencies, and provide it as a dependency
+hosted on the [dep-server](https://github.com/paketo-buildpacks/dep-server).
+
+### Language Family Additions
 
 Using the CycloneDX tool to generate the BOM will involve some changes to the
-NPM Install and Yarn Install Buildpacks.
+Node.JS language family buildpack order groups.
+
+* A single BOM generation buildpack will be added. It's primary functions are:
+  * perform the BOM dependency tool retrieval from the dep-server,
+  * execute BOM generation with the tool (`cyclonedx-bom -o bom.json`), and
+  * contribute the BOM to the layer metadata.
+
+* For each of the three Node.JS buildpack order groups, the BOM generation
+  buildpack will run directly before the start command buildpack. This means:
+  * directly before the `yarn-start` buildpack in the Yarn order group,
+  * directly before the `npm-start` buildpack in the NPM order group, and
+  * directly before the `node-start` buildpack in the no-package-manager order group.
+
+Users of the Node.JS buildpack can opt out of this behaviour by creating a
+custom order grouping that does not include the BOM Generation buildpack, or
+they can set a `ENABLE_MODULE_BOM` flag during container build-time to `false`.
+The default for this flag will be `true`.
+
+#### Detection
+
+The buildpack will pass detection in two different cases:
+
+* Detection will pass if there is a `node_modules` directory in the source directory.
```

Comment:
I don't think apps would have a `node_modules` dir at detect time unless they are vendored. The x-install buildpacks generate node_modules dir during the build phase

Reply:
Yea the point here was that. If the modules are vendored, we don't have to require `node_modules`. My wording is a bit misleading I can fix it up

Reply:
tried to fix the wording in https://github.com/paketo-buildpacks/rfcs/pull/98/commits/7f6e30ec1eedcbb1221f9472bb97fa3464887a54

Reply:
Can we fail detection if BP_*_BOM is set to false? - so we can use it as an optional buildpack. If that's too detailed for the RFC, that's fine too.

Reply:
Yea!! That's what we're proposing. Is that unclear in our explanation?

Reply:
Since we are no longer implementing this environment variable, I don't think this is necessary now?